proto/ociregistry.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package ociregistry;
+
+message OCIActionResultKey {
+  string key_version = 1;
```

Comment:
Can you elaborate on why this key version is needed? That seems unclear.

Reply:
I figured it'd be good to have a way to change the schema of this key and invalidate all previous versions of it. Possibly overkill.

Reply:
It's a useful tool to have, but this can be achieved by changing the "instance name".

Reply:
Good point! Removed the key version field.